README.md

```diff
@@ -125,6 +125,9 @@ See `.trampolinerc` for the canonical list of relevant environment variables.
   include `gs://` or the bucket.
 * `LANGUAGE`: Regenerates all docs under specified language. For example: `LANGUAGE=dotnet`
 * `FORCE_GENERATE_ALL`: Set to `true` to regenerate all docs.
+* `BLOB_TO_DELETE`: Blob to delete from storage. Include full bucket and object name. For
+  example: `gs://my-bucket/docfx-python-test-tarball.tar.gz`
+* `BUILD_LATEST`: Set to `true` to only regenerate latest versions for each package.
```

Comment:
These options feel a bit funny to me:

* `FORCE_GENERATE_ALL` generates everything. Set `BUILD_LATEST` to only do latest.
* `LANGUAGE` generates everything for that language. Set `BUILD_LATEST` to only do latest.

Maybe `LANGUAGE` should the the "optional" item, instead of `BUILD_LATEST`? Then, we could have `FORCE_GENERATE_ALL` and `FORCE_GENERATE_LATEST`, optionally set `LANGUAGE` to restrict to just that language.

Or maybe `BUILD_LATEST` should be named `ONLY_LATEST`? We're still left with the problem it's easy to forget the behavior of `LANGUAGE` -- does that do all for that language?

I guess the question is, what is more intuitive/easier to understand?

1. `FORCE_GENERATE_LATEST=true` `LANGUAGE=foo`
2. `LANGUAGE=foo ONLY_LATEST=true`

I think I like option 1, but I could be convinced.

Reply:
How about if we separate this into a separate Kokoro job, that only builds latest? So instead of `generate-prod`, we'll have `generate-prod-latest` where it'll generate `FORCE_GENERATE_LATEST` by default, and an optional `LANGUAGE=lang` that users can supply if they only want to build latest doc for a language. 

The motivation behind a separate Kokoro job would be to keep the environment variable in Kokoro only to one. The new job can be focused on only building the latest docs, while the existing Kokoro job doesn't have to be modified at all. It has a downside of having to keep track of another Kokoro job for developers.

If we keep this to one Kokoro job, I think both options are equally weighted in my preference. 
```
FORCE_GENERATE_ALL=true, LANGUAGE=lang
FORCE_GENERATE_ALL=true
FORCE_GENERATE_LATEST=true, LANGUAGE=lang
FORCE_GENERATE_LATEST=true
```
or
```
LANGUAGE=lang
LANGUAGE=lang, ONLY_LATEST=true
FORCE_GENERATE_ALL=true
FORCE_GENERATE_ALL=true, ONLY_LATEST=true
```
* Option 1 makes `FORCE_GENERATE_ALL` builds easier, slightly more tacky when you only want to build for single language.
* Option 2 makes `LANGUAGE` builds easier, slightly more tacky when you want to regenerate every latest content.

Reply:
Let's keep a single job for now. If we do parallel builds, we'll make new jobs anyway.

Looking at them now, I think I prefer the first one you gave?

Reply:
Got it.